hummingbird/ml/supported.py

```diff
@@ -339,6 +340,18 @@ def _build_onnxml_operator_list():
     return []
 
 
+def _build_prophet_operator_list():
+    """
+    List all suported XGBoost (Sklearn API) operators.
+    """
+    if prophet_installed():
+        from prophet import Prophet
+
+        return [Prophet]
```

Comment:
I'm confused, is there only one?

Reply:
Yea they only have one operator. (as far as I can tell)